src/components/Agents/Agents.vue

```diff
@@ -465,16 +465,23 @@ export default {
 }
 
 .agent-controls {
-  position: absolute;
-  right: 8px;
-  z-index: 1;
+  left: 50%;
+  position: fixed;
+  transform: translate(570px);
+  z-index: 4;
 
   &.md-and-down {
-    top: -124px;
+    top: 146px;
```

Comment:
changing this to ~120px should fix the overlap

Reply:
Ok should be fixed now, had to add another media query ti capture the sm-md breakpoint.